frontend/src/app/side-bar/side-bar.component.scss

```diff
@@ -1,11 +1,10 @@
-@import '../../assets/_master.scss';
-
+@use "../styles/_variables";
 
 .side-bar {
   height: 100%;
   padding: 139px 51px;
   box-sizing: border-box;
-  border-right: 1px solid $border-color;
+  border-right: 1px solid variables.$gray-color;
```

Comment:
Does it necessary to use 'variables' prefix? Can it be just $gray-color ?

Reply:
done